drizzle/0007_added-emailtokenverification-schema.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE IF NOT EXISTS "EmailVerificationToken" (
+	"id" serial PRIMARY KEY NOT NULL,
+	"token" text NOT NULL,
+	"createdAt" timestamp DEFAULT now() NOT NULL,
+	"expiresAt" timestamp NOT NULL,
+	"email" text NOT NULL,
+	"userId" text NOT NULL,
+	CONSTRAINT "EmailVerificationToken_token_unique" UNIQUE("token")
+);
+--> statement-breakpoint
+DROP INDEX IF EXISTS "BannedUsers_userId_key";--> statement-breakpoint
```

Comment:
Not sure what is going on here because we seem to be deleting and re-adding indexes (I know this is autocreated so I need to test this locally).

Reply:
Yeah even I was not sure why just creating a new schema allows to have a migration which deletes prev indexes and again adding them later